topomodelx/nn/simplicial/sccnn_layer.py

```diff
@@ -207,17 +206,17 @@ def chebyshev_conv(self, conv_operator, conv_order, x):
 
         Parameters
         ----------
-        conv_operator: torch.sparse
-        shape = [n_simplices,n_simplices]
-        e.g., the adjacency matrix, or the Hodge Laplacians
-        conv_order: int
-        the order of the convolution
-        x : torch.Tensor
-        shape = [n_simplices,num_channels]
-
-        Return
+        conv_operator : torch.sparse, shape = [n_simplices,n_simplices]
+            Convolution operator e.g., the adjacency matrix, or the Hodge Laplacians.
+        conv_order : int
+            The order of the convolution.
+        x : torch.Tensor, shape = [n_simplices,num_channels]
+            Feature tensor.
+
+        Returns
         ------
-        x[:, :, k] = (conv_operator@....@conv_operator) @ x
+        X : torch.Tensor
```

Comment:
Single return values do not have a name.

```suggestion
        torch.Tensor
```